### ISSUE TITLE
DNAscent index error

### ISSUE THREAD

**Issue by agouru55** [MBoemo/DNAscent#18]
Hi, whenever running DNAscent index, I get the following error:

```
HDF5-DIAG: Error detected in HDF5 (1.8.14) thread 140515788751616:n21sec  
  #000: H5F.c line 604 in H5Fopen(): unable to open file
    major: File accessibilty
    minor: Unable to open file
  #001: H5Fint.c line 990 in H5F_open(): unable to open file: time = Tue Oct 12 20:05:57 2021
, name = '/path/to/*.fast5', tent_flags = 0
    major: File accessibilty
    minor: Unable to open file
  #002: H5FD.c line 992 in H5FD_open(): open failed
    major: Virtual File Layer
    minor: Unable to initialize object
  #003: H5FDsec2.c line 343 in H5FD_sec2_open(): unable to open file: name = '/path/to/*.fast5', errno = 24, error message = 'Too many open files', flags = 0, o_flags = 0
    major: File accessibilty
    minor: Unable to open file
terminate called after throwing an instance of 'IOerror'
  what():  Could not open file: /path/to/*.fast5
Aborted (core dumped)
```

**Comment by MBoemo** [comment]
Okay no problem.  Please can I just confirm 1.) you're not using the suffix "/*.fast5" in the directory you are specifying, as that will give you a list of fast5 files rather than a directory, and 2.) you have read permissions for all of these files?

**Comment by MBoemo** [comment]
Closing for now and assuming this was fixed as of [commit 76c5aca](https://github.com/MBoemo/DNAscent/commit/76c5acaa567e68416fec4c883bd49536094229f9) but reopen if there are further issues.

**Comment by MBoemo** [comment]
I tested this on a few different datasets this morning and wasn't able to reproduce the issue, so it looks like everything's working as expected. This type of error is thrown when DNAscent can't find the fast5 file that corresponds to a particular sequence alignment in the bam file.  There could be a few different reasons for this:
- you moved the fast5 files since building the index,
- there's at least one read in the bam file that's not in the index which is usually caused by not passing the top-level directory containing the fast5 files for this run,
- when calling `DNAscent index` you passed a relative path to the fast5 files instead of the full path.

You could also try printing the first few lines of the index file to make sure everything looks okay, as these are human-readable and might point to what the problem is.  After a brief header, index files follow a two-column format with readID in the left column and the full path to the corresponding fast5 file in the right column.
